src/main/resources/twitter-users/component.xml

```diff
@@ -20,71 +20,85 @@
     <subComponents>
```

Comment:
Shall we have a display Name in the component level as well? So that we can have seperate sections in the UI?